index.html

```diff
@@ -172,24 +172,24 @@ <h3 style="margin: 0px 40%">But Why Us?</h3>
         <div class="col-md-4">
           <div class="card p-3 py-4 mt-5 mb-5">
             <div class="text-center">
-              <img src="./siddhi.PNG" width="100" class="rounded-circle" />
+              <img src="./siddhi.PNG" alt="siddhi's profile icon" width="100" class="rounded-circle" />
               <h3 class="mt-2">Siddhi Bhanushali</h3>
               <span class="mt-1 clearfix">Android Developer</span>
               <small class="mt-4"
                 >I am an Android Developer working at Google, California,
                 USA</small
               >
               <div class="social-buttons mt-5">
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's twitter handle">
                   <i class="fa-brands fa-twitter"></i>
                 </button>
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's linkedin handle">
                   <i class="fa-brands fa-linkedin-in"></i>
                 </button>
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's instagram handle">
                   <i class="fa-brands fa-instagram"></i>
                 </button>
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's github handle">
```

Comment:
@muditchoudhary , its not correctly spelled

Reply:
ohh...I misspelled it, let me fix it

Reply:
> @muditchoudhary , its not correctly spelled

Updated, see if everything is correct